src/rules/declaration-block-no-duplicate-properties/README.md

```diff
@@ -62,6 +62,40 @@ p {
 }
 ```
 
+### `ignore: ["consecutive-duplicates-with-different-values"]`
+
+Ignore consecutive duplicated properties with different walues.
+
+They can prove to be useful fallbacks for older browsers but only make sense if their values are not equal.
+
+The following patterns are considered warnings:
+
+```css
+p {
+  font-size: 16px;  
+  font-size: 16px;
+  font-weight: 400;
+}
+```
+
+```css
+p {
+  font-size: 16px;
+  font-weight: 400;
+  font-size: 1rem;
```

Comment:
Why would this be a warning? Those are different values, as far as we're able to tell (we have no idea what the font-size is on the `<html>` element).